src/main/java/com/redhat/devtools/lsp4ij/LSPIJUtils.java

```diff
@@ -759,11 +769,12 @@ public static Range toRange(TextRange range, Document document) {
     public static @Nullable TextRange toTextRange(@NotNull Range range,
```

Comment:
And this signature can/should be `private` since it's only used from `LSPIJUtils`.